etc/org.opencastproject.kernel.security.LtiLaunchAuthenticationHandler.cfg

```diff
@@ -0,0 +1,42 @@
+# OAuth consumer keys with should be highly trusted.
+#
+# By default OAuth consumer are regarded as untrusted and a user authenticating via such
+# systems receives a rewritten username in the form of "lit:{ltiConsumerGUID}:{ltiUserID}".
+# This user is regarded as a new user temporarily existing for the duration of the session.
+# Opencast roles associated with the original user will not be attached to this user.
+#
+# Usernames of users authenticating via highly trusted systems will not be rewritten except
+# for the cases configured in the additional options below.
+#
+# Multiple consumer keys can be configured, by incrementing the counter. The list is read
+# sequentially incrementing the counter. If you miss any numbers it will stop looking for
+# further consumer keys.
+#lti.oauth.highly_trusted_consumer_key.1=CONSUMERKEY
+
+# Allow the Opencast system administrator user to authenticate as such via LTI.
+#
+# Note that this user may still authenticate via LTI, but the username will be rewritten,
+# even if a trusted OAuth consumer key is used.
+#
+# Note that this option does not apply to custom users having the ROLE_ADMIN. Use the
+# blacklist below instead.
+#
+# Default: false
+#lti.allow_system_administrator=true
+
+# Allow the Opencast digest user to authenticate as such via LTI.
+#
+# Note that this user may still authenticate via LTI, but the username will be rewritten,
+# even if a trusted OAuth consumer key is used.
+#
+# Default: false
+#lti.allow_digest_user=true
+
+# A blacklist of users not allowed to authenticate via LTI as themselves.
+#
+# Note that these users may still authenticate via LTI, but their username will be rewritten,
+# even if a trusted OAuth consumer key is used.
```

Comment:
I feel like I'm not 100% sure what this paragraph means.  They can't authenticate as themselves via LTI, but their username is rewritten, as it is in the other cases as well.  Maybe I'm just not familiar enough with how LTI works to get this, but it seems like this description needs to be fleshed out more.

Reply:
Well their username is rewritten (e.g. `lti:lmsng.school.edu:1234` instead of `mtneug`). Opencast's role provider will see the new username and therefore do not load roles associated with `mtneug`. So more accurately: they can't authenticate as themselves because their username is rewritten.

Reply:
Ok, so this is down to my not having worked with LTI.  We're good here.